src/WhiteLibrary/keywords/items/listcontrols.py

```diff
@@ -45,6 +46,44 @@ def listbox_selection_should_be(self, locator, expected):
             raise AssertionError("Expected listbox selection to be " +   # noqa: W504
                                  expected + ", was " + listbox.SelectedItemText)
 
+    @keyword
+    def listbox_should_contain(self, locator, expected):
+        """Checks that listbox contains an element.
```

Comment:
This could be clarified: "..  listbox contains an item with text `expected`"
Also use "item" instead of "element", so the documentation of the new keywords uses the same word as other keyword docstrings.